app/models/investigation.rb

```diff
@@ -1,5 +1,6 @@
 class Investigation < ApplicationRecord
   include Seek::Rdf::RdfGeneration
+  include Seek::Taggable
```

Comment:
since all Investigation, Study and Assay now supports tagging, this can be moved into `acts_as_isa` ( same for `Study`)

Reply:
incidentally, the same could be said for the existing `include Seek::Rdf::RdfGeneration` and that could be moved too.

Reply:
Resolved in [c77b9c6](https://github.com/seek4science/seek/pull/1350/commits/c77b9c6ab7da4bf566374f5aed70470f56ac28c9).
Moved both RdfGeneration and Taggable to act_as_isa.